src/app/design/guidelines/content-containers/index.html

```diff
@@ -7,15 +7,29 @@
     Divide a page into sections of related content
```

Comment:
I know this is existing content, but it might be worth double-checking with Todd to see if tiles are only for pages. That's how everything is written here, but I assume tiles can also be used in modals.

Reply:
Checked in with Todd about this. He said you can use tiles in modals, but ideally an alternate approach would be used instead.

Reply:
OK. Sounds like we should leave the references to "page" only throughout. Thanks for checking!